### ISSUE TITLE
[Usable: Designer_ Card Element]: All the buttons under "Card Elements" section are not accessible with the mouse.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7410]
https://microsoft.visualstudio.com/OS/_workitems/edit/30926087
